docker-compose.yml

```diff
@@ -140,6 +140,7 @@ services:
       - gitlab-config:/etc/gitlab
       - gitlab-logs:/var/log/gitlab
       - gitlab-data:/var/opt/gitlab
+      - ./services/gitlab/Auto-DevOps.gitlab-ci.yml:/opt/gitlab/embedded/service/gitlab-rails/vendor/gitlab-ci-yml/Auto-DevOps.gitlab-ci.yml:ro
```

Comment:
@rokroskar do you see anything smelly here? Why does it fail on travis?

Reply:
it also fails on my machine